File: Java/백준/Bronze/2576. 홀수/홀수.java
import java.util.Scanner;

public class Main {
    public static void main (String[] args) {
        Scanner sc = new Scanner (System.in);
        
        int sum = 0;
        int min_val = 101;
        
        for (int i = 0; i < 7; i++) {
            int num = sc.nextInt();
            
            if (num % 2 != 0) {
                sum += num;
                
                if (num < min_val) {
                    min_val = num;
                }
            }
        }
        
        if (sum == 0) {
            System.out.println(-1);
        } else {
            System.out.println(sum);
            System.out.println(min_val);
        }
        
        sc.close();
    }
}

File: C++17/백준/Bronze/2576. 홀수/홀수.cc
#include <iostream>
using namespace std;

int main() {
    int min_val = 101;
    int sum = 0;
    
    for (int i = 0; i < 7; i++) {
        int n;
        cin >> n;
        
        if (n % 2 != 0) { 
            sum += n;
            
            if (n < min_val) {
                min_val = n;
            }
        }
    }
    
    if (sum == 0) {
        cout << "-1" << "\n";
    } else {
        cout << sum << "\n";
        cout << min_val << "\n";
    }
    
    return 0;
}